pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -160,6 +160,11 @@ type CertificateSpec struct {
 	// Options to control private keys used for the Certificate.
 	// +optional
 	PrivateKey *CertificatePrivateKey `json:"privateKey,omitempty"`
+
+	// EncodeUsagesInRequest controls whether key usages should be present
+	// in the CertificateRequest
+	// +optional
+	EncodeUsagesInRequest bool `json:"encode_usages_in_request,omitempty"`
```

Comment:
using snake case is against the Kubernetes style guide

Reply:
So just everything stuck together?

Reply:
Ah no, camel case, just saw.